src/main/groovy/net/wasdev/wlp/gradle/plugins/Liberty.groovy

```diff
@@ -163,4 +208,12 @@ class Liberty implements Plugin<Project> {
             task.server = project.liberty.server
         }
     }
+
+    String installAppsDependsOn(ServerExtension server, String elseDepends) {
+        if (server.apps != null || server.dropins != null) {
```

Comment:
Double check that server.apps and server.dropins are not initialized as an empty list and never null.